src/mergeHead.ts

```diff
@@ -14,6 +14,10 @@ const getOriginalHeadObject = (that, originalHead): JsonLDObject => {
 };
 
 const getJsonLdHeadObject = (that, jsonLdFunc: Function, space: Options['space']): JsonLDObject => {
+  if (!jsonLdFunc) {
+    return null;
+  }
```

Comment:
This line checks existence of jsonld function instead of https://github.com/ymmooot/nuxt-jsonld/pull/478/files#diff-8f208d7224ba2e70eccc891041744eeb2f5a986d1cfb969df0be7a975308c48fL21